hypervisor/arch/x86/cpu.c

```diff
@@ -77,6 +77,12 @@ bool is_pcpu_active(uint16_t pcpu_id)
 {
 	return bitmap_test(pcpu_id, &pcpu_active_bitmap);
 }
+
+uint64_t get_pcpu_active_bitmap(void)
+{
```

Comment:
From English p.o.v., This should be get_active_pcpu_bitmap...

Reply:
ok, will update